gazebo_plugins/CMakeLists.txt

```diff
@@ -14,12 +14,30 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(gazebo_dev REQUIRED)
 find_package(gazebo_ros REQUIRED)
+find_package(geometry_msgs REQUIRED)
 find_package(rclcpp REQUIRED)
 
 include_directories(include
   ${gazebo_ros_INCLUDE_DIRS}
+  ${geometry_msgs_INCLUDE_DIRS}
 )
 
+# gazebo_ros_force
+add_library(gazebo_ros_force SHARED
+  src/gazebo_ros_force.cpp
+)
+target_link_libraries(gazebo_ros_force
+  ${gazebo_ros_LIBRARIES}
+  ${geometry_msgs_LIBRARIES}
+)
```

Comment:
Want to make sure these lines and the include_directories lines are needed

Reply:
It seems to work ok without the link libs now. I know I originally added them for a reason... Weird... The include still seems to be needed, but removed libs: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/784/commits/0cc0e06f4aaca8b9cabd025dde907402afc9fc7a